### ISSUE TITLE
Image streaming with webcam is broken

### ISSUE THREAD

**Issue by aliabd** [gradio-app/gradio#6271]
### Describe the bug

Image streaming using webcam as a source is broken. It returns 

`ValueError: Image streaming only available if sources is ['webcam']. Streaming not supported with multiple sources.`

even though source is only webcam:

` gr.Image(sources=["webcam"], streaming=True)`

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

import gradio as gr
import numpy as np

def flip(im):
    return np.flipud(im)

demo = gr.Interface(
    flip, 
    gr.Image(sources=["webcam"], streaming=True), 
    "image",
    live=True
)
if __name__ == "__main__":
    demo.launch()
    

### Logs

```shell
File "/Users/aliabdalla/Desktop/dev/gradio/gradio/components/image.py", line 122, in __init__
    raise ValueError(
ValueError: Image streaming only available if sources is ['webcam']. Streaming not supported with multiple sources.
```

### System Info

```shell
gradio 4.0.2
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Thanks @aliabd let me push a quick fix

**Comment by abidlabs** [comment]
Closed via #6272
